test/encapsulation/browser.js

```diff
@@ -1,9 +1,11 @@
-var getStyle = require('computed-style');
-var test = require('tape-css')(require('tape'));
-var styles = require('../fixtures/encapsulation.out.css');
+'use strict';
 
-var dom = (function () {
-  var container = document.createElement('div');
+const getStyle = require('computed-style');
+const test = require('tape-css')(require('tape'));
+const styles = require('../fixtures/encapsulation.out.css');
+
+const dom = ((() => {
```

Comment:
the additional expression to wrap the iife is probably unnecessary `(() => {})()` should be enough